fitforge-convex-test/client/src/components/muscle-heatmap/MuscleHeatMap.tsx

```
@@ -0,0 +1,337 @@
+
+      {/* Muscle Detail Tooltip */}
+      {selectedMuscle && (
+        <MuscleTooltip
+          muscle={selectedMuscle}
+          data={recoveryStates.find(s => s.muscleGroup === selectedMuscle) || null}
+          onClose={() => setSelectedMuscle(null)}
+        />
+      )}
+
+      {/* Hover Tooltip */}
+      {hoveredMuscle && hoveredMuscle.data && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)